radiotools/atmosphere/refractivity.py

```diff
@@ -35,7 +39,7 @@ def n_param_ZHAireS(h):
     return 1 + a * np.exp(-b * h)
 
 
-def get_refractivity_between_two_points_numerical(p1, p2, atm_model=None, refractivity_at_sea_level=None, table=None, debug=False):
+def get_refractivity_between_two_points_numerical(p1, p2, atm_model=None, refractivity_at_sea_level=None, table=None):
```

Comment:
I you simply remove this option, it will break user code.

Reply:
Keep it, but maybe print a message, if someone actively tries to set this to true.